libcodechecker/analyze/log_parser.py

```diff
@@ -67,9 +80,30 @@ def get_compiler_defines(compiler):
                                     stderr=subprocess.PIPE)
             out, err = proc.communicate("")
             for line in out.splitlines(True):
-                LOG.debug("define:"+line)
+                LOG.debug("define: " + line)
                 define = line.strip().split(" ")[1:]
-                d = "-D"+define[0] + '=' + '"' + ' '.join(define[1:]) + '"'
+
+                variable = define[0]
+                value = ' '.join(define[1:])
+
+                if variable in IGNORED_DEFINES:
```

Comment:
This should be the other way around, as `__has_include_next` must match `__has_include_next(STR)`, not swapped.